### PR TITLE
[release] prepare 0.39.0 release

### PR DESCRIPTION
**Pull request by codeboten**
- update changelog
- run make multi-prerelease
- run make gotidy

### CODE REVIEW

**Review by bogdandrutu — APPROVED** [review]

File: cmd/configschema/go.mod [hunk 1]
```diff
@@ -5,8 +5,8 @@ go 1.17
 require (
 	github.com/fatih/structtag v1.2.0
 	github.com/google/uuid v1.3.0
-	github.com/open-telemetry/opentelemetry-collector-contrib v0.38.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/resourcetotelemetry v0.38.0
+	github.com/open-telemetry/opentelemetry-collector-contrib v0.39.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/resourcetotelemetry v0.39.0
 	github.com/stretchr/testify v1.7.0
 	go.opentelemetry.io/collector v0.37.1-0.20211026180946-46c8e2290e45
 	go.opentelemetry.io/collector/model v0.37.1-0.20211026180946-46c8e2290e45
```
Comment (Aneurysm9) [inline cmd/configschema/go.mod:12]: Shouldn't these be updated to v0.39.0?

**Review by Aneurysm9 — APPROVED** [review]

File: exporter/jaegerthrifthttpexporter/go.sum [hunk 2]
```diff
@@ -799,19 +775,14 @@ go.opencensus.io v0.22.4/go.mod h1:yxeiOL68Rb0Xd1ddK5vPZ/oVn4vY4Ynel7k9FzqtOIw=
 go.opencensus.io v0.22.5/go.mod h1:5pWMHQbX5EPX2/62yrJeAkowc+lfs/XD7Uxpq3pI6kk=
 go.opencensus.io v0.23.0 h1:gqCw0LfLxScz8irSi8exQc7fyQ0fKQU/qnC/X8+V/1M=
 go.opencensus.io v0.23.0/go.mod h1:XItmlyltB5F7CS4xOC1DcqMoFqwtC6OG2xF7mCv7P7E=
-go.opentelemetry.io/collector v0.38.0/go.mod h1:z3DoFBVBO4eRjDTxR3FptJeSdFXWv+OPlyb2vpjOP2k=
 go.opentelemetry.io/collector v0.39.0 h1:SZqzUuK7Lmlf4o2GRTBIntuTJNs9WN+8XGZ6yWILGGM=
 go.opentelemetry.io/collector v0.39.0/go.mod h1:LSOk+VL3LFHE6Q8wuWeljbRZYpnx36wJH807vDElcos=
-go.opentelemetry.io/collector/model v0.36.0/go.mod h1:+7YCSjJG+MqiIFjauzt7oM2qkqBsaJWh5hcsO4fwsAc=
 go.opentelemetry.io/collector/model v0.38.0/go.mod h1:gS8A27wi+8gM3hrXL+dEjTbrbLxktjHjAwwqI31ELgQ=
```
Comment (jcchavezs) [inline exporter/jaegerthrifthttpexporter/go.sum:780]: Curious about why this is still needed.

File: exporter/jaegerthrifthttpexporter/go.sum [hunk 3]
```diff
@@ -799,19 +775,14 @@ go.opencensus.io v0.22.4/go.mod h1:yxeiOL68Rb0Xd1ddK5vPZ/oVn4vY4Ynel7k9FzqtOIw=
...
 go.opentelemetry.io/collector/model v0.39.0 h1:5kWA6B5Sa6qjFlp1QtuvUgsG8TNaMZ1kBdS5QiF9nLQ=
 go.opentelemetry.io/collector/model v0.39.0/go.mod h1:gS8A27wi+8gM3hrXL+dEjTbrbLxktjHjAwwqI31ELgQ=
-go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.25.0/go.mod h1:E5NNboN0UqSAki0Atn9kVwaN7I+l25gGxDqBueo/74E=
 go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.26.1/go.mod h1:4wsfAAW5N9wUHM0QTmZS8z7fvYZ1rv3m+sVeSpf8NhU=
-go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.25.0/go.mod h1:NyB05cd+yPX6W5SiRNuJ90w7PV2+g2cgRbsPL7MvpME=
 go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.26.1 h1:/PDcqsmxpbI/3ERJ6s6cwF13ZSH5m9NNCOPsoeazEhA=
```
Comment (jcchavezs) [inline exporter/jaegerthrifthttpexporter/go.sum:784]: Same for this one. This looks very old.

Comment (codeboten) [reply exporter/jaegerthrifthttpexporter/go.sum:784]: This is a version from 2 weeks ago, not sure if that's what you mean when you say very old. https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp@v0.26.1

File: exporter/jaegerthrifthttpexporter/go.sum (hunk 2 above)
Comment (codeboten) [reply exporter/jaegerthrifthttpexporter/go.sum:780]: It's a good question, appears to be the same for other exporters. I won't have time to dig into it for this release though.

File: exporter/jaegerthrifthttpexporter/go.sum (hunk 3 above)
Comment (jcchavezs) [reply exporter/jaegerthrifthttpexporter/go.sum:784]: Forget about it. This is for opentelemetry-go not for contrib.

File: cmd/configschema/go.mod (hunk 1 above)
Comment (codeboten) [reply cmd/configschema/go.mod]: Updated.

File: exporter/jaegerthrifthttpexporter/go.sum (hunk 2 above)
Comment (jcchavezs) [reply exporter/jaegerthrifthttpexporter/go.sum:780]: I fixed it in this PR https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/6317

**Pull request merged**